invenio_oauthclient/handlers.py

```diff
@@ -234,20 +235,18 @@ def authorized_signup_handler(resp, remote, *args, **kwargs):
 
         if user is None:
             # Auto sign-up if user not found
-            user = oauth_register(account_info)
-            if user is None:
```

Comment:
@jirikuncar I don't think so, since then it will always display the sign-up form, instead of just register a user if all information is available?